### PR TITLE
Pool: fixes the invalid access control in the pool contract

### PR DESCRIPTION
**Pull request by gangov**

### CODE REVIEW

File: contracts/pool/src/tests/config.rs [hunk 1]
```diff
@@ -95,10 +94,9 @@ fn update_config() {
 }
 
 #[test]
-#[should_panic(expected = "Pool: UpdateConfig: Unauthorize")]
+#[should_panic]
```
Comment (ueco-jb) [inline contracts/pool/src/tests/config.rs:97]: Why did you remove the panic message?

Comment (gangov) [reply contracts/pool/src/tests/config.rs]: once that we got rid of the `sender` parameter, I thought that it's okay just to expect panic. Fixed now, as per your comments from here and yesterday

**Review by ueco-jb — APPROVED** [review]
LGTM

File: contracts/pool/src/tests/config.rs (hunk 1 above)
Comment (ueco-jb) [reply contracts/pool/src/tests/config.rs]: You should never skip the message, because your implementation might panic for whole variety of reasons.
